bit/find_missing_number.py

```diff
@@ -0,0 +1,15 @@
+def find_missing_number(nums):
+    """Returns the missing number from a sorted list of unique
```

Comment:
Since, we are doing xor operations here, I don't think it is required for the input list to be sorted, is it?
The code should work for a general list of integers in range [0..n], where one number is missing and the difference between any two numbers is 1.